src/redisenterprise/azext_redisenterprise/aaz/latest/redisenterprise/_update.py

```diff
@@ -102,7 +102,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.user_assigned_identity_resource_id = AAZStrArg(
             options=["--identity-resource-id", "--user-assigned-identity-resource-id"],
             arg_group="KeyEncryptionKeyIdentity",
-            help="User assigned identity to use for accessing key encryption key Url. Ex: `/subscriptions/<sub uuid>/resourceGroups/<resource group>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myId`.",
+            help="User assigned identity to use for accessing key encryption key Url. Ex: /subscriptions/<sub uuid>/resourceGroups/<resource group>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myId.",
             nullable=True,
```

Comment:
Parameter: redisenterprise update, `user_assigned_identity_resource_id` - Disallowed html tags ['<sub uuid>', '<resource group>'] in short summary

Reply:
BTW, if developer has changed the aaz model, please commit the model change in aaz repo as well, so that future developer won't lost the adjustment previous developer has made already.